app/wama.web/Views/Patron/Create.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@{
```

Comment:
data doesnt looks like getting lost, it remains in filled even error is shown as far as i testing it